R/add.R

```diff
@@ -94,19 +97,24 @@ add_insulin_purchases_cols <- function(gld_purchases) {
 #' @param data Data from [include_diabetes_diagnoses()] function.
 #'
 #' @returns The same type as the input data, default as a [tibble::tibble()],
-#'  with the following columns and up to two rows per individual:
+#'  with the following added columns and up to two rows per individual:
 #'
-#'  -   `pnr`: The personal identification variable.
-#'  -   `dates`: The dates of the first and second hospital diabetes diagnosis.
 #'  -   `has_majority_t1d_diagnosis`: A logical vector indicating whether the
```

Comment:
```suggestion
#'  -   `has_majority_t1d_diagnoses`: A logical vector indicating whether the
```
Maybe? :) If yes, then the logic should be renamed as well.